src/Logging.AzureAppServices/Logging.AzureAppServices.slnf

```diff
@@ -0,0 +1,9 @@
+{
+  "solution": {
+    "path": "..\\..\\AspNetCore.sln",
+    "projects": [
+      "src\\Logging.AzureAppServices\\src\\Microsoft.Extensions.Logging.AzureAppServices.csproj",
```

Comment:
I doubt this is correct. If you run clean.cmd then open this project in VS it will need a lot more projects I'm assuming.

Reply:
It seems correct to me. This is an extensions package that only depend on extensions packages from dotnet/runtime.

Reply:
Oh duh, extensions isn't in this repo